### PR TITLE
feat(benchmark): rename the benchmarks from `{revision}` and `PNPM` to `pacquet@{revision}` and `pnpm`

### PR DESCRIPTION
**Pull request by KSXGitHub**

### CODE REVIEW

File: tasks/integrated-benchmark/src/work_env.rs [hunk 1]
```diff
@@ -26,17 +27,21 @@ pub struct WorkEnv {
...
     fn root(&self) -> &'_ Path {
         &self.root
     }
 
-    fn revisions(&self) -> impl Iterator<Item = &'_ str> + '_ {
+    fn revision_names(&self) -> impl Iterator<Item = &'_ str> + '_ {
         self.revisions.iter().map(AsRef::as_ref)
     }
 
+    fn revision_subs(&self) -> impl Iterator<Item = SubDir<'_>> + '_ {
```
Comment (zkochan) [inline tasks/integrated-benchmark/src/work_env.rs:41]: what does revision_subs mean? revision_sub_dirs?

File: tasks/integrated-benchmark/src/work_env.rs [hunk 2]
```diff
@@ -26,17 +27,21 @@ pub struct WorkEnv {
...
+        self.revisions.iter().map(AsRef::as_ref).map(SubDir::PacquetRevision)
```
Comment (zkochan) [inline tasks/integrated-benchmark/src/work_env.rs:42]: can't this be refactored to
```suggestion
        self.revision_names().map(SubDir::PacquetRevision)
```

File: tasks/integrated-benchmark/src/work_env.rs [hunk 3]
```diff
@@ -45,16 +50,16 @@ impl WorkEnv {
...
 
-    fn revision_root(&self, revision: &str) -> PathBuf {
-        self.root().join(revision)
+    fn sub_dir_path(&self, sub_dir: SubDir) -> PathBuf {
+        self.root().join(sub_dir.to_string())
     }
 
-    fn revision_install_script(&self, revision: &str) -> PathBuf {
-        self.revision_root(revision).join("install.bash")
+    fn sub_install_script(&self, sub_dir: SubDir) -> PathBuf {
```
Comment (zkochan) [inline tasks/integrated-benchmark/src/work_env.rs:57]: what does sub_install_script mean? does it mean `install_script_in_sub_dir`?

**Review by zkochan — COMMENTED** [review]
sub_dir doesn't sound like a better name than revision or revision_dir. Was it renamed because pnpm has a dir too? If I understand correctly the sub_dirs are where install runs. Maybe call it `bench_dir`, or `test_dir`, or `action_dir`?

File: tasks/integrated-benchmark/src/work_env.rs (hunk 3 above)
Comment (KSXGitHub) [reply tasks/integrated-benchmark/src/work_env.rs:57]: Actually, I think just `script` will be a better name.

**Comment by KSXGitHub** [comment]
> Was it renamed because pnpm has a dir too?

Yes.

File: tasks/integrated-benchmark/src/work_env.rs (hunk 3 above)
Comment (KSXGitHub) [reply tasks/integrated-benchmark/src/work_env.rs:57]: > `install_script_in_sub_dir`

This name makes it look like it's installing some script in some directory, while in fact, it's just getting the path to an `install.bash` script.

File: tasks/integrated-benchmark/src/work_env.rs (hunk 1 above)
Comment (KSXGitHub) [reply tasks/integrated-benchmark/src/work_env.rs]: This method has been renamed to `revision_ids`.

The `SubDir` struct has also been renamed to `BenchId`.

File: tasks/integrated-benchmark/src/work_env.rs (hunk 3 above)
Comment (KSXGitHub) [reply tasks/integrated-benchmark/src/work_env.rs]: It has been renamed to `script_path`.

**Review by zkochan — APPROVED** [review]

**Pull request merged**